tests/functional/aws-node-sdk/test/bucket/putACL.js

```diff
@@ -1,9 +1,64 @@
 import assert from 'assert';
+import { S3 } from 'aws-sdk';
 
 import withV4 from '../support/withV4';
 import BucketUtility from '../../lib/utility/bucket-util';
+import getConfig from '../support/config';
 
+const bucket = `bucketacl-bucket-${Date.now()}`;
 const bucketName = 'putbucketaclfttest';
```

Comment:
is it intentional to use different bucket names for the different tests or could you use the same var for both?